static/js/util/test_utils.js

```diff
@@ -21,14 +22,31 @@ export function findCourse(courseSelector: (course: Course, program: Program) =>
   throw "Unable to find course";
 }
 
-export const alterFirstRun = (course: Course, overrideObject: Object): Course => {
+export const alterFirstRun = (course: Course, overrideObject: Object): CourseRun => {
   course.runs[0] = Object.assign({}, course.runs[0], overrideObject);
+  return course.runs[0];
 };
 
 export function findAndCloneCourse(courseSelector: (course: Course, program: Program) => boolean): Course {
   return _.cloneDeep(findCourse(courseSelector));
 }
 
+export function setCourseRunCount(course: Course, desiredRunCount: number, courseRunToCopy: ?CourseRun) {
+  let currentRunCount = course.runs.length;
+  if (currentRunCount < desiredRunCount) {
+    let courseRun = currentRunCount === 0 ? courseRunToCopy : course.runs[0];
+    if (!courseRun) {
+      throw new Error('Need a course run to copy.');
+    }
+    let runsNeeded = desiredRunCount - currentRunCount;
+    for (let i = 0; i < runsNeeded; i++) {
+      course.runs.push(_.cloneDeep(courseRun));
```

Comment:
When we're creating course runs can you modify the course run so that it's unique? We should make sure the `id` and `course_id` are unique and `position` is unique for that particular course